Library/Homebrew/global.rb

```diff
@@ -30,7 +30,7 @@
 HOMEBREW_USER_AGENT_CURL = ENV["HOMEBREW_USER_AGENT_CURL"]
 HOMEBREW_USER_AGENT_RUBY = "#{ENV["HOMEBREW_USER_AGENT"]} ruby/#{RUBY_VERSION}-p#{RUBY_PATCHLEVEL}"
```

Comment:
It seems that `HOMEBREW_USER_AGENT_RUBY` is no longer used anywhere in the code.